src/Common/src/Common.Kubernetes/Steeltoe.Common.Kubernetes.csproj

```diff
@@ -1,14 +1,18 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <Description>Steeltoe common library for Kubernetes</Description>
     <PackageTags>Kubernetes</PackageTags>
   </PropertyGroup>
 
   <Import Project="..\..\..\..\versions.props" />
   <Import Project="..\..\..\..\sharedproject.props" />
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
```

Comment:
Intent is there should be no changes for netstandard2.0 since this is a patch release... We won't be testing the netstandard2.0 path (but shouldn't have anything to worry about as there's no change for that scenario) and we will drop this target in a future major/minor release